### PR TITLE
LUN-305: Swipe gesture to delete individual area disfunctional

### PR DESCRIPTION
**Pull request by f1sh1918**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

Check Steps to reproduce in https://issues.tuerantuer.org/browse/LUN-305

Since this happens mainly on ios pls check on ios emulator or real device

@klinzo i'm not 100% sure if the test adjustments make sense :)

### CODE REVIEW

File: src/components/__tests__/ListItem.spec.tsx [hunk 1]
```diff
@@ -67,14 +67,14 @@ describe('ListItem', () => {
     })
   })
 
-  it('should not call on press callback if scrolling', async () => {
+  it('should not call on press callback if scrolling or swiping', async () => {
```
Comment (klinzo) [inline src/components/__tests__/ListItem.spec.tsx:70]: I think two separate tests would make sense here, one that scrolling (only change y position) and one that swiping(only change x position) does not trigger on click. On the other hand the swipe to delete functionality is removed in #270 anyway. But I think just copying and adjusting would be easy and won't hurt.

Comment (f1sh1918) [reply src/components/__tests__/ListItem.spec.tsx:70]: hm ok i overseen the fact that it gonna be removed. it we don't make any hotfix release. this pr is maybe not needed at all. So i won't put any more effort in tests to be honest

**Review by klinzo — APPROVED** [review]
Tested on browserstack

**Review by SteffiStuffel — CHANGES_REQUESTED** [review]
We don't need this changes. Next release will contain the individualized home screen and there won't be this swipe-to-delete component anymore.

**Comment by f1sh1918** [comment]
> We don't need this changes. Next release will contain the individualized home screen and there won't be this swipe-to-delete component anymore.

yes i realized this way too late. gonna close the pr

**Pull request closed without merging**